packages/open-next/src/core/patchAsyncStorage.ts

```diff
@@ -35,5 +35,5 @@ export function patchAsyncStorage() {
       );
 
     // We use `bind` here to avoid referencing outside variables to create potential memory leaks.
-  }.bind(null, resolveFilename);
+  }).bind(null, resolveFilename);
```

Comment:
I know that bind work differently for `this` for arrow function vs normal ones. I guess it would not have any effect here, not entirely sure. If there is no effect then it's fine

Reply:
If `bind` is not used to set the value of `this`, it can be used to pre-supply parameters. Here’s an example: https://stackoverflow.com/a/50630093

Reply:
Yeah i know that, i have a different concern here, in the stackoverflow answer they said this 
> Partially, because bind returns a new function that calls the original function with a specific this — but arrow functions don't use the this they're called with, they completely ignore it

That's the part i'm concerned about, this **could** mean that if the underlying function use `this` it might have a completely different behaviour than expected. Not sure though

Reply:
If it seems unsafe, we could disable the `useConst` rule globally or only for these cases.

I can make these changes in a separate PR tomorrow, focusing solely on biome-related modifications. This will make it easier to handle `.git-blame-ignore-revs`. After merging, you could then rebase this PR.

Reply:
This is not a global things, these 2 function are very specific as their goal is to override existing functions.
I will not be able to test this today, if you'd wish to open a PR, maybe try focusing on stuff not covered in this PR, there is still a lot of warning https://github.com/opennextjs/opennextjs-aws/actions/runs/11746705545/job/32727036460?pr=627, no need to do the same job twice

Reply:
But:
- do we want to update the comment / add warning
- report an issue to biome (this is an auto-fix)

Reply:
> do we want to update the comment / add warning

I guess we can, but this is such a weird edge case that i hope we don't have to do this anytime soon 

> report an issue to biome (this is an auto-fix)

The behavior in biome is correct actually, if we add a `this` inside the function then biome does not report an error anymore. 
There might be an issue if we used the function directly instead of through `.call`, but since it is used to monkey patch a node built-in, this is probably not something that is covered (It should probably even be it's own rule, like `dontOverrideNodeBuiltIn`